azure_data_cosmos_engine/query-tests/src/runner.rs

```diff
@@ -196,13 +196,13 @@ fn create_client() -> Result<CosmosClient, azure_core::Error> {
             .danger_accept_invalid_certs(true)
             .build()
             .map_err(|e| {
-                azure_core::Error::full(
+                azure_core::Error::with_error(
                     azure_core::error::ErrorKind::Other,
                     e,
                     "failed to create HTTP client",
                 )
             })?;
-        options.client_options.transport = Some(TransportOptions::new(Arc::new(http_client)))
+        options.client_options.transport = Some(Transport::new(Arc::new(http_client)))
```

Comment:
was this just a change by Core in the naming of an object that effectively does the same thing?

Reply:
Yep!